test/Servant/QuickCheck/InternalSpec.hs

```diff
@@ -135,7 +134,7 @@ type API = ReqBody '[JSON] String :> Post '[JSON] String
 api :: Proxy API
 api = Proxy
 
-type ParamsAPI = QueryParam "one" String :> QueryParam "two" String :> Get '[JSON] String
+type ParamsAPI = QueryParam "one" String :> QueryParam "two" String :> Get '[JSON] ()
```

Comment:
Riiiiight. That's better too - I'll update the PR.

In fact I've realised that the bug also applies to `QueryFlag`s so will replicate for those (almost identical)